javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,143 @@
+             /********************************************************/
+               console.log('\n        Smallest letter in Array\n\n')
+             /********************************************************/      
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const smalleLetter=getSmallestLetter(danishWords); // returns 'ø'
+function getSmallestLetter(words){
+    let temp=danishWords[0];
+    for(let i=0;i<danishWords.length;i++){
+         if(danishWords[i].length<=temp.length){
+             temp=danishWords[i];
+             danishWords[i]=danishWords[0];
+             danishWords[0]=temp;
+
+         }
+
+    }
+    return danishWords[0];
+    //console.log(danishWords[0])
+
+}
+document.getElementById('smallestLetter').innerHTML='Smallest letter in array : '+smalleLetter;
+console.log('Smallest letter in array ["bil", "plante", "kaffe", "bog", "ø", "planetarium"] is : '+smalleLetter);
+
+
+                /********************************************************/
+                      console.log('\n        Danish letters count\n\n')
+                /********************************************************/ 
+
+const danishString2 = "Blå grød med røde bær";
+let danishLetters=danishLettersCounter(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1}
```

Comment:
you function does not return `{total: 4, æ: 1, ø: 2, å: 1}` but a string. Also if a letter has the zero count, you list it, where the solution should not be listing letters with zero count

Reply:
done..returning object now